src/ee/tthk/tarpv19/basharov/IpValidatorTests.java

```diff
@@ -48,21 +48,21 @@ void ShouldBe_False_GivenStringWithFourNumbersNotInRange0_255(){
 	@Test
 	void ShouldBe_False_GivenIPstartingWithZero() {
 		IpValidator validator= new IpValidator();
-		assertFalse(validator.ValidateIpv4Address("0.2.3.4"));
+		assertTrue(validator.ValidateIpv4Address("0.2.3.4"));
 	}
 	@Test
 	void ShouldBe_False_GivenIPendingWithZero() {
 		IpValidator validator= new IpValidator();
-		assertFalse(validator.ValidateIpv4Address("1.2.3.0"));
+		assertTrue(validator.ValidateIpv4Address("1.2.3.0"));
 	}
 	@Test
 	void ShouldBe_False_GivenIPstartingWith255() {
 		IpValidator validator= new IpValidator();
-		assertFalse(validator.ValidateIpv4Address("255.2.3.4"));
+		assertTrue(validator.ValidateIpv4Address("255.2.3.4"));
```

Comment:
Test name should be FALSE, but assertion is TRUE.

To pass the test you don't change the test. You change the code under test. In this case ValidateIpv4Address method.